CorsixTH/Src/th_gfx.cpp

```diff
@@ -1332,6 +1334,7 @@ animation::animation()
   hit_test_fn = THAnimation_hit_test;
   is_multiple_frame_animation_fn = THAnimation_is_multiple_frame_animation;
   patient_effect = animation_effect::none;
```

Comment:
Several things here that probably belong in the initialization list - but none of them new in this PR. Just an observation.